client/settings/express-checkout/index.js

```diff
@@ -176,6 +200,29 @@ const ExpressCheckout = () => {
 							</a>
 						</div>
 					</li>
+					{ displayLinkPaymentMethod && (
+						<li className="express-checkout has-icon-border">
+							<div className="express-checkout__checkbox loadable-checkbox label-hidden">
+								<CheckboxControl
+									label={ 'Stripe Link' }
+									checked={ isStripeLinkEnabled }
+									onChange={ updateStripeLinkCheckout }
+								/>
+								<div className="express-checkout__icon">
```

Comment:
I believe a designer can better judge here, but I think having a padding around the logo looks better:

Before:
<img width="702" alt="Screen Shot 2022-04-15 at 12 25 55" src="https://user-images.githubusercontent.com/12542046/163570469-a64cc350-aba2-4ec7-9265-dd6ba8dd1867.png">

After:
<img width="695" alt="Screen Shot 2022-04-15 at 10 32 09" src="https://user-images.githubusercontent.com/12542046/163570479-3b297424-fe7a-4ec0-9862-2c75a5e7ab9a.png">

Also, having StipeLink without a description looks a little "empty". We may need to add a small description there.